e2e/app-with-domain/app_with_domain_test.go

```diff
@@ -102,6 +102,86 @@ var _ = Describe("App With Domain", func() {
 				Expect(err).NotTo(HaveOccurred())
 			}
 		})
+
+		It("should create environment manifests", func() {
+			Expect("./copilot/environments/test/manifest.yml").Should(BeAnExistingFile())
+			Expect("./copilot/environments/prod/manifest.yml").Should(BeAnExistingFile())
+		})
+
+		It("should show both environments in env ls", func() {
+			envList, envListError := cli.EnvList(appName)
+			Expect(envListError).NotTo(HaveOccurred())
+			Expect(len(envList.Envs)).To(Equal(2))
+		})
+
+		It("should show only bootstrap resources in env show", func() {
```

Comment:
To simplify the e2e tests maybe we just need this case for one of the test suites 🤔

Reply:
done!